src/library/dao/playlistdao.h

```diff
@@ -40,6 +40,12 @@ class PlaylistDAO : public QObject, public virtual DAO {
         PLHT_UNKNOWN = -1
     };
 
+    enum class AutoDJSendLoc {
+        TOP,
+        BOTTOM,
+        REPLACE,
+    };
+
```

Comment:
I would not touch the track, loaded on the deck. This could be an action out of sight. 
Removing the top track from Auto DJ will remove the loaded track anyway if the user enables AutoDJ.